### PR TITLE
[#1317] LCO: replaceable EMF Forms

### LINKED ISSUE #1317: Operator: replaceable EMF Forms
LCO extensively uses EMF Forms.
Let grow them replacable by extension code.

### PR DESCRIPTION
**Pull request by eparovyshnaya**
Closes #1317

### CODE REVIEW

File: bundles/org.eclipse.passage.loc.api/src/org/eclipse/passage/loc/internal/api/LocObjectMentorProperty.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package org.eclipse.passage.loc.internal.api;
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.api/src/org/eclipse/passage/loc/internal/api/LocObjectMentorProperty.java:1]: header

File: bundles/org.eclipse.passage.loc.api/src/org/eclipse/passage/loc/internal/api/LocObjectMentorProperty.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+import java.util.function.Supplier;
+
+public final class LocObjectMentorProperty implements Supplier<String> {
+
+	@Override
+	public String get() {
+		return "locMentor"; //$NON-NLS-1$
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.api/src/org/eclipse/passage/loc/internal/api/LocObjectMentorProperty.java:9]: `"mentor"` ?

File: bundles/org.eclipse.passage.loc.api/src/org/eclipse/passage/loc/internal/api/LocObjectMentorshipService.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package org.eclipse.passage.loc.internal.api;
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.api/src/org/eclipse/passage/loc/internal/api/LocObjectMentorshipService.java:1]: header

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/WithMentor.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import org.eclipse.passage.loc.internal.api.LocObjectMentorProperty;
+import org.eclipse.passage.loc.internal.api.LocObjectMentorshipService;
+
+public final class WithMentor {
+
+	private final EObject target;
+	private final IEclipseContext context;
+
+	public WithMentor(EObject target, IEclipseContext context) {
+		this.target = target;
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/WithMentor.java:17]: welcome `null` checks?

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/WithMentor.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
+package org.eclipse.passage.loc.dashboard.ui.wizards;
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/WithMentor.java:1]: header

**Review by ruspl-afed — CHANGES_REQUESTED** [review]
missed headers

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/WithMentor.java (hunk 4 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/WithMentor.java:17]: added

File: bundles/org.eclipse.passage.loc.api/src/org/eclipse/passage/loc/internal/api/LocObjectMentorshipService.java (hunk 3 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.api/src/org/eclipse/passage/loc/internal/api/LocObjectMentorshipService.java:1]: added

File: bundles/org.eclipse.passage.loc.api/src/org/eclipse/passage/loc/internal/api/LocObjectMentorProperty.java (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.api/src/org/eclipse/passage/loc/internal/api/LocObjectMentorProperty.java:1]: added

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/WithMentor.java (hunk 5 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/WithMentor.java:1]: added

**Review by ruspl-afed — APPROVED** [review]
LGTM

**Pull request merged**